src/App/App.tsx

```diff
@@ -28,6 +28,7 @@ const StripePaymentStatus = lazy(() => import("pages/StripePaymentStatus"));
 const Widget = lazy(() => import("pages/Widget"));
 const BlogPosts = lazy(() => import("pages/Blog/Posts"));
 const BlogPost = lazy(() => import("pages/Blog/Post"));
+const Home = lazy(() => import("pages/Home/Homepage/Home"));
```

Comment:
```suggestion
const Home = lazy(() => import("pages/Home/Home"));
```
`HomePage` is extra folder, 

could be 
```suggestion
const Home = lazy(() => import("pages/Home"));
```

if you add `Home/index.ts` which handles export
```
//Home/index.ts
export {default} from "./Home.tsx"
```